templates/output.tpl

```diff
@@ -13,4 +13,4 @@ ${ ansible ? "To rerun the Ansible provisioner using your `${ssh_key}` private k
 
     ansible-playbook playbook.yml -v -i ./.ansible/hosts -e '@./.ansible/extra_vars.json'" : "" }
 
-Thank you and enjoy using CircleCI Enterprise!
+Thank you and enjoy using CircleCI Server!
```

Comment:
Given Nathan'c comments in #release we should probably remove Server and just have CircleCI... cuz... you know... bizdev....

Reply:
I thought that we were explicitly supposed to say "Server". Im so confused lol. @ndintenfass ?

Reply:
I just read his thing. that motivates me. Will change.